onnx_chainer/export.py

```diff
@@ -407,6 +413,8 @@ def _export(model, args, filename, export_params, graph_name, save_text,
     o = Graph(context, converters, opset_version, network_outputs)
     o.to_onnx_graph()
 
+    hook.restore_inputs()
```

Comment:
[optional] I think it'd be better to use `__exit__` and make the scope of `hook` larger since, ideally, this should be done even when an exception is thrown. I don't think we should do this in this PR, though.

Reply:
I agree, fixed